### PR TITLE
[CARBONDATA-386] Unit test case for CarbonMetadataUtil,CarbonMergerUtil,DataFileFooterConverter,DataTypeUtil

### PR DESCRIPTION
**Pull request by abhisheknoldus**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [x] Make sure the PR title is formatted like:
   `[CARBONDATA-<Jira issue #>] Description of pull request`
 - [x] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [x] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [x] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).
 - [ ] Testing done
 
        Please provide details on 
        - Whether new unit test cases have been added or why no new tests are required?
        - What manual testing you have done?
        - Any additional information to help reviewers in testing this change.
         
 - [ ] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA. 
                 

---

### CODE REVIEW

File: core/src/test/java/org/apache/carbondata/core/util/CarbonMetadataUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static junit.framework.TestCase.*;
+import static org.apache.carbondata.core.util.CarbonMetadataUtil.getBlockIndexInfo;
+import static org.apache.carbondata.core.util.CarbonMetadataUtil.getIndexHeader;
+
+public class CarbonMetadataUtilTest {
+
```
Comment (ravipesala) [inline core/src/test/java/org/apache/carbondata/core/util/CarbonMetadataUtilTest.java]: There are many methods to cover in CarbonMetadataUtil

**Comment by ravipesala** [comment]
Please format the classes

**Comment by abhisheknoldus** [comment]
@ravipesala I have added the test cases for CarbonMetadataUtil class and format all the classes.
Please check.

**Comment by ravipesala** [comment]
LGTM

**Pull request closed without merging**